recipe/meta.yaml

```diff
@@ -51,6 +51,7 @@ requirements:
     - ipp-include
     - ipp-devel
     - ipp
+    - cvxpy
```

Comment:
This should be a test requirement, not a build requirement.

Reply:
fixed